document log level override option for mcp logs (#8227)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-ai/setup-local-mcp.md

- [Cursor](/docs/dbt-ai/integrate-mcp-cursor)
- [VS Code](/docs/dbt-ai/integrate-mcp-vscode)

> ## Debug configs

Reviewer: ```suggestion
## Debug configurations
These settings allow you to customize the MCP server’s logging level to help with diagnosing and troubleshooting.

```

| Name                     | Default | Description                                                                     |
| ------------------------ | ------- | ------------------------------------------------------------------------------- |

> | `DBT_MCP_LOG_LEVEL`        | `INFO` | Override the MCP server log level. Options: DEBUG, INFO, WARNING, ERROR, CRITICAL  |

Reviewer: sugg being clear that they're env var
```suggestion
| `DBT_MCP_LOG_LEVEL`        | `INFO` |  Environment variable to override the MCP server log level. Options are: `DEBUG`, `INFO`, `WARNING`, `ERROR`, `CRITICAL`.  |
```

> | `DBT_MCP_LOG_LEVEL`        | `INFO` | Override the MCP server log level. Options: DEBUG, INFO, WARNING, ERROR, CRITICAL  |

Reviewer: @reneruck-dbt not critical but what are the differences in terms of logs for each option? e.g. is `debug` more detailed and itemized?

Author: Debug logs are usually much more  and more detailed. They tend to give a much more verbose picture of what's happening under the hood.

> A good way to find out what is happening inside the MCP server and debug issues is to increase the log level to `DEBUG`. 

Reviewer: ```suggestion
To see more detail about what’s happening inside the MCP server and help debug issues, you can set the log level to `DEBUG`.
```

> A good way to find out what is happening inside the MCP server and debug issues is to increase the log level to `DEBUG`. 

Reviewer: @reneruck-dbt do we want to recommend users update the env var to `debug` permanently or temporarily? or it doesn't matter?

Author: We would recommend to only update temporarily while investigating an issue. Otherwise debug logs will unnecessarily fill up disk space with logs.

- [Cursor](/docs/dbt-ai/integrate-mcp-cursor)
- [VS Code](/docs/dbt-ai/integrate-mcp-vscode)

## Debug configurations
These settings allow you to customize the MCP server’s logging level to help with diagnosing and troubleshooting.

| Name                     | Default | Description                                                                     |
| ------------------------ | ------- | ------------------------------------------------------------------------------- |
| `DBT_MCP_LOG_LEVEL`        | `INFO` |  Environment variable to override the MCP server log level. Options are: `DEBUG`, `INFO`, `WARNING`, `ERROR`, `CRITICAL`.  |

> To see more detail about what’s happening inside the MCP server and help debug issues, you can set the log level to `DEBUG`.

Reviewer: ```suggestion
To see more detail about what’s happening inside the MCP server and help debug issues, you can temporarily set the log level to `DEBUG`.
```

> To see more detail about what’s happening inside the MCP server and help debug issues, you can temporarily set the log level to `DEBUG`.

Reviewer: ```suggestion
To see more detail about what’s happening inside the MCP server and help debug issues, you can temporarily set the log level to `DEBUG`. We recommend setting it temporarily to avoid filling up disk space with logs.
```
